cross-chain/arbitrum/deploy_l2/21_transfer_arbitrum_tbtc_ownership.ts

```diff
@@ -0,0 +1,14 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, helpers } = hre
+  const { deployer, governance } = await getNamedAccounts()
+
+  await helpers.ownable.transferOwnership("ArbitrumTBTC", governance, deployer)
+}
+
+export default func
+
+func.tags = ["TransferArbitrumTBTCOwnership"]
+func.dependencies = ["ArbitrumTBTC"]
```

Comment:
Should we add `func.runAtTheEnd = true`? 

I think it is possible now that this step will run right after `ArbitrumTBTC` making the gateway authorization impossible.

Reply:
I guess it won't hurt, but the indexing should do the trick [ee35831](https://github.com/keep-network/tbtc-v2/pull/554/commits/ee35831e3b903d0dc5143ea51e11ca94a2bba6e8)

Reply:
Do you mean the prefixes for file names? I could be wrong but I do not think they matter at all. What matters for establishing an order in which scripts are executed is what we have in `func.dependencies` and `func.runAtTheEnd`.

Reply:
Based on this docs https://github.com/wighawag/hardhat-deploy#the-deploy-task my understanding was that they will be executed in alphabetical order (in our case in ascending).

> It will scan for files in alphabetical order and execute them in turn.

Reply:
Maybe we should add `AuthorizeWormholeGateway` as a dependency to make sure the transfer is executed after the authorization?